texttospeech/ssml_addresses/README.md

```diff
@@ -0,0 +1,28 @@
+# Google Cloud Text-to-Speech API
```

Comment:
Typically in this repo, we auto-generate our README using the following script. 

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/scripts/readme-gen/readme_gen.py

It takes a README.rst.in file similar to this one: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/texttospeech/cloud-client/README.rst.in

Reply:
Thanks! Could you point me to where I can find instructions on how to run the script?

Reply:
`python path_to_script/readme_gen.py path_to_readme/README.rst.in`

Reply:
Thanks!